av/container/core.pyx

```diff
@@ -10,6 +11,9 @@ from av.container.output cimport OutputContainer
 from av.container.pyio cimport pyio_read, pyio_write, pyio_seek
 from av.format cimport build_container_format
 from av.utils cimport err_check, dict_to_avdict
+from posix.time cimport gettimeofday
+from av.container.core cimport cb_info
```

Comment:
Keep the imports sorted alpbatically, and regroup the imports from the same module